tests/e2e/specs/table.cy.js

```diff
@@ -124,3 +124,54 @@ describe('Table view', () => {
     })
   })
 })
+
+function addView (view) {
+  cy.get('[data-cy=add-view-btn]').click()
+  cy.get(`#toolbar-add-${view}-view`).click()
+    // wait for menu to close
+    .should('not.be.exist')
+}
+
```

Comment:
I started a new `describe.` as - i think - the...
```
beforeEach(() => {
    cy.visit('/#/table/one')
  })
```
associated with the last one was stopping me from navigating away and back again